src/test/scenarios/managed-network/configuration/readme.az.md

```diff
@@ -58,6 +59,24 @@ cli:
             op: CreateOrUpdate
             param: properties
         poly-resource: true
+      - where:
+            group: managedNetworkPeeringPolicies
+            op: Delete
+        max-api: '2020-09-01'
+        min-api: '2019-09-01'
+        # resource-type: DATA_COMPUTE   # enable this after clicommon support it
```

Comment:
Does this mean clicommon haven't support it yet?

Reply:
not support yet,  I have a PR here for it: https://github.com/Azure/autorest.clicommon/pull/101